src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -557,7 +558,7 @@ case class WorkflowActor(workflow: WorkflowDescriptor, backend: Backend)
       case backendCallKey: BackendCallKey => fetchLocallyQualifiedInputs(backendCallKey) match {
         case Success(callInputs) => startBackendCallWithMessage(message, backendCallKey, callInputs)
         case Failure(t) =>
-          self ! CallFailedToInitialize(callKey)
+          self ! CallFailedToInitialize(callKey, s"Failed to fetch locally qualified inputs: ${t.getMessage}")
```

Comment:
hmm that might incur the wrath of @geoffjentry; I think it might be best to leave this as is